Backend/src/v1/run_ml.py

```diff
@@ -11,26 +11,29 @@
     tags=["Run ML"],
 )
 
-
 @router.get("/", status_code=200)
-def run(model_name: str, file: str) -> dict:
+def run(model_name: str, file: str, dim_red_method: str) -> dict:
```

Comment:
If you set `dim_red_method: str | None = None`  you can run without giving a value, which I think makes more sense especially if running the test_model. 

If all methods **needs** to have a dim_red_method, I think it might be a good thing to specify a default value here instead of using `=None`. This would also remove the need to fill with a  dummy value when running the test model.

Reply:
Perfect, I was not sure about how to implement that, thanks for the tip